lib/module.js

```diff
@@ -144,52 +182,24 @@ Module._findPath = function(request, paths, isMain) {
     return Module._pathCache[cacheKey];
   }
 
-  var exts;
+  const exts = Object.keys(Module._extensions);
```

Comment:
Simplified the control flow, since `tryFindPath` was split out in order to reduce redundancy (just like `readFilePath`).

More than happy to add it back in, but seemed like a very marginal cost to me.